tokenserver/tests/test_service.py

```diff
@@ -163,6 +163,19 @@ def test_unauthorized_error_status(self):
             with self.assertRaises(ValueError):
                 res = self.app.get('/1.0/sync/1.1', headers=headers)
 
+    def test_unverified_token(self):
+        headers = {'Authorization': 'BrowserID %s' % self._getassertion()}
+        mock_response = {
+            "status": "okay",
+            "email": "test@mozilla.com",
+            "idpClaims": {
+                "fxa-tokenVerified": False
+            }
+        }
+        with self.mock_verifier(response=mock_response):
+            res = self.app.get("/1.0/sync/1.1", headers=headers, status=401)
+        self.assertEqual(res.json['status'], 'invalid-credentials')
+
```

Comment:
I think we should also have a test for the explicit `fxa-tokenVerified = True` case.